packages/cactus-plugin-ledger-connector-quorum/src/main/typescript/plugin-ledger-connector-quorum.ts

```diff
@@ -654,4 +711,67 @@ export class PluginLedgerConnectorQuorum
     }
     return this.runDeploy(req);
   }
+
+  // Low level function to call any method from web3.eth
+  // Should be used only if given functionality is not already covered by another endpoint.
+  public async invokeRawWeb3EthMethod(
+    args: InvokeRawWeb3EthMethodV1Request,
+  ): Promise<any> {
+    this.log.debug("invokeRawWeb3EthMethod input:", JSON.stringify(args));
+
+    Checks.nonBlankString(
+      args.methodName,
+      "web3.eth method string must not be empty",
+    );
+
+    const looseWeb3Eth = this.web3.eth as any;
+    if (
+      !Object.prototype.hasOwnProperty.call(looseWeb3Eth, args.methodName) ||
+      typeof looseWeb3Eth[args.methodName] !== "function"
+    ) {
+      throw new Error(`No method "${args.methodName}" in web3.eth`);
+    }
+
+    const web3MethodArgs = args.params || [];
+    return looseWeb3Eth[args.methodName](...web3MethodArgs);
+  }
+
+  // Low level function to invoke contract
+  // Should be used only if given functionality is not already covered by another endpoint.
+  public async invokeRawWeb3EthContract(
+    args: InvokeRawWeb3EthContractV1Request,
+  ): Promise<any> {
+    this.log.debug("invokeRawWeb3EthContract input:", JSON.stringify(args));
+
+    const contractMethodArgs = args.contractMethodArgs || [];
+
+    if (
+      !Object.values(EthContractInvocationWeb3Method).includes(
+        args.invocationType,
+      )
+    ) {
+      throw new Error(
+        `Unknown invocationType (${args.invocationType}), must be specified in EthContractInvocationWeb3Method`,
+      );
+    }
+
+    const contract = new this.web3.eth.Contract(
+      args.abi as AbiItem[],
+      args.address,
+    );
+
+    const isSafeToCall = await this.isSafeToCallContractMethod(
+      contract,
+      args.contractMethod,
+    );
+    if (!isSafeToCall) {
+      throw new RuntimeError(
+        `Invalid method name provided in request. ${args.contractMethod} does not exist on the Web3 contract object's "methods" property.`,
+      );
+    }
+
+    return contract.methods[args.contractMethod](...contractMethodArgs)[
```

Comment:
@outSH I'll dismiss this one as a false-positive because the `isSafeToCallContractMethod` call is right above it so AFAICT this is all good.